packages/ipfs-topology/src/ipfs-topology.ts

```diff
@@ -74,6 +74,15 @@ export class IpfsTopology {
     readonly logger: DiagnosticsLogger,
     readonly period: number = DEFAULT_PEER_DISCOVERY_PERIOD
   ) {}
+  
+  /**
+   * Connect to custom peer list
+   * @param peerList List of multiaddresses to swarm connect to
+   */
+  async connect(peerList: Array<string>) {
```

Comment:
is this function meant to be used somewhere?

Reply:
+1 to this.